_posts/2022-03-21-ugrc-web-services-api-funding.md

```diff
@@ -0,0 +1,51 @@
+
+In the early days of UGRC, the Web API was not a known technology; now it has become the de facto standard for enhancing and providing access to the SGID data. UGRC does not have a current budget for this effort. For years UGRC developed the technology with funds from: other projects that utilize the API, appropriation funding, and small enhancement request funding from other agencies and organizations that are interested in supporting the API. However, none of these undedicated sources are sufficient to properly enhance, update, and maintain the UGRC Web API system.
+
+As agencies continually move toward data-driven approaches to serve the citizens of Utah, location data must be consistently robust, repeatable, and easily accessed by problem solvers so they can make the most relevant data-driven decisions. To meet the increasing demand for consistent statewide data, UGRC needs consistent, increased funding. This increased funding would support several web services, including Next Generation 911 (NG911), lieutenant governor voting efforts and administrative boundary certification, Medicaid fraud detection, treasurer’s office unclaimed property section, fleet management, streamlined sales tax, and other programs that require a citizen location to provide the appropriate services.
+
+
+Why UGRC Web Services? 
+UGRC web services require funding because they support so many other government applications, including the following services:
+
+NG911: Improved routing of emergency calls to a public safety answering point and dispatch of emergency vehicles
+
+Voting Applications: Assignment of voters to precincts, and precincts to districts 
+
+Detection of Medicaid and other types of fraud
+
+Assistance with the Watershed Restoration Initiative 
+
+Additional web services funding will allow UGRC to make required changes and improvements that are needed in order to handle the widespread increase in demand for these centralized UGRC services. 
+
+UGRC has used existing resources to develop web services to their current state and have tried to continue on this path; however, the significant adoption and reliance on the UGRC Web API requires a much more focused and attentive approach, as monthly requests to the API have risen in excess of twelve million. UGRC’s need to generate revenue by providing cost-recovery project work greatly limits the time we are able to put toward the Web API. We are simply not able to maintain it like we should, such as updating dependencies and implementing new technologies to ensure its robustness. These standard IT practices should be happening regularly; however, the last major update to this enterprise service was over five years ago.
```

Comment:
**thought**: i think we need to be careful with our wording here so we don't make people question the accuracy etc of the service.

Reply:
I will try to reword this